cloud-img/README.md

```diff
@@ -0,0 +1,18 @@
+# Custom Image for Stackdriver Sandbox Project
+
+When the Dockerfile changes, here are the steps to follow in order to push it to GCR:
+
+Build the Dockerfile, tag the image, and push the image to the Container Registry.
+
+```bash
+$ docker build -t cloudshell-image .
+
+$ gcloud auth configure-docker 
```

Comment:
This is command you need to run only once.

Reply:
Good point. I will clarify that in the documentation.